src/main/java/org/logstash/beats/Server.java

```diff
@@ -34,25 +31,22 @@
 
     private final int clientInactivityTimeoutSeconds;
 
-    public Server(String host, int p) {
-        this(host, p, DEFAULT_CLIENT_TIMEOUT_SECONDS, Runtime.getRuntime().availableProcessors() * 4);
-    }
-
-    public Server(String host, int p, int timeout, int threadCount) {
+   public Server(String host, int p, int timeout, int threadCount) {
+//       ResourceLeakDetector.setLevel(ResourceLeakDetector.Level.PARANOID);
         this.host = host;
         port = p;
         clientInactivityTimeoutSeconds = timeout;
         beatsHeandlerThreadCount = threadCount;
         workGroup = new NioEventLoopGroup();
-    }
+   }
 
-    public void enableSSL(SslSimpleBuilder builder) {
+    void enableSSL(SslSimpleBuilder builder) {
```

Comment:
@robbavey any particular reason for changing the permissions here?

Reply:
Just trying to remove unnecessarily permissive access modifiers. Will revert these changes and follow up later.